libs/ui/src/hand_marked_paper_ballot_prose.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { HandMarkedPaperBallotProse } from './hand_marked_paper_ballot_prose';
+
+const proseContent = (
+  <React.Fragment>
+    <hr />
+    <h3>Heading 3</h3>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor omnis quam
+      modi magnam neque. Molestias recusandae, officia maiores nam pariatur
+      earum qui inventore minus enim adipisci nemo voluptate at harum?
+    </p>
+    <h4>Heading 4</h4>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor omnis quam
+      modi magnam neque. Molestias recusandae, officia maiores nam pariatur
+      earum qui inventore minus enim adipisci nemo voluptate at harum?
+    </p>
+    <h5>Heading 5</h5>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor omnis quam
+      modi magnam neque. Molestias recusandae, officia maiores nam pariatur
+      earum qui inventore minus enim adipisci nemo voluptate at harum?
+    </p>
+    <hr />
+  </React.Fragment>
+);
+describe('renders HandMarkedPaperBallotProse', () => {
+  test('with defaults', () => {
+    const { container } = render(
+      <HandMarkedPaperBallotProse>{proseContent}</HandMarkedPaperBallotProse>
+    );
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
I looked through these snapshots and I couldn't tell the difference - they seem to be identical. Is it possible to add more specific test assertions for the differences between these cases instead of using snapshots?

Reply:
Replaced all the snapshots for the `Main` component with verification of logic: https://github.com/votingworks/vxsuite/pull/1819/files#diff-f57a44aa306887ba87f92fa4265dfcaec9d6858fa45ae9ab8dcf4a6a8c2fe1b5

Will apply here next week.

Reply:
Just pushed fresh test code.